src/talondoc/sphinx/domains.py

```diff
@@ -15,14 +14,16 @@
 
 _LOGGER = getLogger(__name__)
 
+__talonDirectiveClasses__ = TalonActionDirective | TalonCaptureDirective
```

Comment:
Why was this added?

Reply:
Not sure about that one. I think it might have been for a rule that ended up getting removed, but I am not certian. Let me just revert that one real quick.

Reply:
Done